source/ways-of-working/standards/pull-requests.html.md.erb

```diff
@@ -130,6 +130,19 @@ All repositories in the organisation will automatically use this by default.
 
 You should start with this and adopt for your needs as needed by copying it into repository in the `.github` folder and editing it.
 
+### Suggested Branch Protection Rules
+
+These are the recommended settings to configure for your main branch on your repository, however each project might run differently so these are not These are the recommended settings to configure for your main branch on your repository, however each project might run differently so these are not compulsory.
+Check and fill out the below options on the Branch Protection Rules of your main branch.
+
+- Require a pull request before merging
+    - Require Approvals (minimal 1)
+    - Dismiss stale pull request approvals when new commits are pushed
+- Require status checks to pass before merging
+    - Require branches to be up to date before merging <br/>
```

Comment:
the time wasted by requiring this is silly, the number of times this will cause you an issue is close to 0 but will cost you far more to run with this, instead you can enable the button always allow updating branch from main settings page not branch protection so people can optionally update if they want to
```suggestion
```

Reply:
mistake on my part adding this one.